### PR TITLE
Add image pull secrets to deployments, fix extra pod labels whitespace

### PR DESCRIPTION
**Pull request by waakanni**
added image pull secret to all remaining charts, removed empty line from template and bumped up the version

### CODE REVIEW

File: charts/auth-km-jobs/templates/cronjob.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ spec:
             {{- end }}
         spec:
           serviceAccountName: {{ .Values.serviceAccount.name }}
+          {{- if .Values.imagePullSecretNames }}
```
Comment (ckaipf) [inline charts/auth-km-jobs/templates/cronjob.yml:24]: This is a `job` did you verify that this is a correct object spec (it should be OK)?

Comment (waakanni) [reply charts/auth-km-jobs/templates/cronjob.yml:24]: We spoke about this previously and you said I should add it. It renders properly but I didn't try to install it

Comment (ckaipf) [reply charts/auth-km-jobs/templates/cronjob.yml:24]: Ok, great, thanks. I just wanted to be sure that is doesn't break anything.

**Comment by waakanni** [comment]
> Wasn't merged @waakanni needs some updates.

I have bumped up the version but it keeps failing

**Pull request merged**